lib/logger/bindLogger.js

```diff
@@ -0,0 +1,66 @@
+const { performance } = require('perf_hooks');
+const { getDatadogSecret } = require('./getDatadogSecret');
+const { initLogger } = require('./logger');
+const { initDataDogMetricsApi } = require('./metricsApiWrapper');
+
+const initDependencies = async () => {
+	const logVersion = 'v1';
+	if (process.env.FALAFEL_ENABLE_LOGGING !== 'true') {
+		return { metricsApiWrapper: undefined, logger: undefined };
+	}
+	try {
+		const apiKey = await getDatadogSecret(process.env.DATADOG_SSM_PARAM_KEY, process.env.AWS_REGION);
+		const metricsApiWrapper = await initDataDogMetricsApi(process.env.DATADOG_HOST, apiKey, logVersion);
+		const logger = initLogger(process.env.DATADOG_HOST, apiKey, logVersion);
+		return { metricsApiWrapper, logger };
+	} catch (error) {
+		if (process.env.FALAFEL_ENABLE_LOGGING === 'true') {
```

Comment:
this if statement is redundant, as the execution flow would not come here if logging was not enabled.